src/library/module.cpp

```diff
@@ -490,18 +488,33 @@ environment add_inductive(environment                       env,
 }
 } // end of namespace module
 
+bool is_candidate_olean_file(std::string const & file_name) {
+    std::ifstream in(file_name);
+    deserializer d1(in, optional<std::string>(file_name));
+    std::string header, version;
+    d1 >> header;
+    if (header != g_olean_header)
+        return false;
+    d1 >> version;
+#ifndef LEAN_IGNORE_OLEAN_VERSION
```

Comment:
We should probably add a platform-generic make readme where we can document such config variables

Reply:
I agree.